lib/siwapp_web/controllers/page_controller.ex

```diff
@@ -14,12 +15,18 @@ defmodule SiwappWeb.PageController do
     invoice =
       id
       |> String.to_integer()
-      |> Invoices.get!()
+      |> Invoices.get()
```

Comment:
The problem of doing that is that I don't know if all cases that error should be dispatch. I'll verify it

Reply:
In case of `get!` that's an Elixir convention that means that an error is going to be raised if something goes wrong so it should be valid. The thing is to be consistent in all contexts.

Reply:
I looked if this method where use in API and they aren't. If them where used What would happen raising an error which  is meant to be used as frontal error?